src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -36,6 +36,10 @@ def get_initial_value(self, submission: Submission | None = None) -> str:
             return ""
         return submission.public_registration_reference
 
+    @staticmethod
+    def as_json_schema():
+        raise NotImplementedError()
```

Comment:
this is done because the base class defines it as `@abstractmethod` and you *must* implement it, right?

I think we should provide a default implementation instead of crashing, so drop the `@abstractmethod` and instead use the minimal schema generation from `get_json_schema_for_user_defined_variable`